alvr/filesystem/src/lib.rs

```diff
@@ -0,0 +1,200 @@
+use std::{
+    env,
+    path::{Path, PathBuf},
+};
+
+#[cfg(target_os = "linux")]
+const SERVER_BUILD_DIR_NAME: &str = "alvr_server_linux";
+#[cfg(windows)]
+const SERVER_BUILD_DIR_NAME: &str = "alvr_server_windows";
+#[cfg(target_os = "macos")]
+const SERVER_BUILD_DIR_NAME: &str = "alvr_server_macos";
```

Comment:
You should download Steam and check if it matches the name they call it, for consistency's sake.

Reply:
This is dummy, there is no macos support in SteamVR. I just want to compile it on my machine. If you don't like dummy values then I can reuse Linux constants.

Reply:
I prefer to keep this to differentiate the directories in case I want to cross-build for linux.

Reply:
> If you don't like dummy values then I can reuse Linux constants

I do not like them but I'd prefer you checked what they call it in `.steam/root` folder. For me it is `linux64` but I don't think Valve will call it MacOS since Steam has been around for longer than that, maybe darwin?

Reply:
I'll check.